composer.json

```diff
@@ -65,7 +65,7 @@
     "oat-sa/generis" : ">=14.0.0",
     "oat-sa/tao-core" : ">=48.9.0",

```

Comment:
I can see that one of the original PRs [requires later versions](https://github.com/oat-sa/extension-tao-testqti/pull/2117/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R65-R66) of generis and core.

I suppose that bump was to ensure the DI container features are available.
Could you please double check those dependencies' bump?

Reply:
@wazelin, agreed. This indeed was forgotten, as I didn't cherry-pick commits with composer files - too much to merge.
As I can see in november release we, in any case, have the needed versions.
Adding changes to this PR with needed version bump in a minute